src/runtime/entitlements-manager.js

```diff
@@ -183,20 +184,12 @@ export class EntitlementsManager {
    * @private
    */
   showToast_(entitlement) {
-    const toast = new Toast(this.win_, {
-      text:
-          (entitlement.source || 'google') == 'google' ?
-          'Access via Google Subscriptions' :
-          // TODO(dvoytenko): display name instead.
-          'Access via [' + entitlement.source + ']',
-      action: {
-        label: 'View',
-        handler: function() {
-          // TODO(dparikh): Implementation.
-        },
-      },
-    });
-    toast.open();
+    const source = entitlement.source || 'google';
+
+    return new Toast(this.deps_, feUrl('/toastiframe'), feArgs({
+      'publicationId': this.publicationId_,
```

Comment:
Hmm. Is there such a variable in this class? I don't recall and not seeing it added. Perhaps something also worth to cover in tests?

Reply:
Updated.

Reply:
One request. There are uses in this class with `this.config_.getPublicationId()`. Since you added this as a variable, please either switch to this variable, or use `this.config_.getPublicationId()` everywhere.

Reply:
Found only one instance, replaced with `this.publicationId_`